### ISSUE TITLE
No module named MorseAnalysis

### ISSUE THREAD

**Issue by marijazmf** [AppliedMathematicsANU/diamorse#2]
Dear all,

I installed diamorse successfully, but  when I tried this step:
./python/persistence.py -t 1.0 -r tomo_floatimage_IMP_SEDT.nc > pairs.txt

I got error message:

Traceback (most recent call last):
  File "./python/persistence.py", line 4, in <module>
    from MorseAnalysis import VolumeImage, VectorField
ImportError: No module named MorseAnalysis

Any recommendation, what can be wrong?

Thanks in advance.
Regards,
Marija

**Comment by odf** [comment]
Hi Marija,

The problem is that the installation process creates a file `MorseAnalysis.so` in `./bin`, but Python expects it in `./python`. In the past, I have recommended setting the `PYTHONPATH` environment variable accordingly, but an easier solution is to copy the file over or create a symbolic link. I've just checked in a slightly modified `./src/python/Makefile` that creates that link automatically upon installation. Hopefully, this will solve the issue.

Cheers,
Olaf
